packages/rrweb/src/replay/index.ts

```diff
@@ -481,18 +491,22 @@ export class Replayer {
     }
     if (typeof config.mouseTail !== 'undefined') {
       if (config.mouseTail === false) {
-        if (this.mouseTail) {
-          this.mouseTail.style.display = 'none';
+        for (const [, { mouseTail }] of Object.entries(this.pointers)) {
+          if (mouseTail) {
+            mouseTail.style.display = 'none';
+          }
```

Comment:
Small nit, since you're only using the value, you could use `Object.values` instead of `entries`.